src/main/java/console_app/ApplicationDriver.java

```diff
@@ -93,50 +92,45 @@ private boolean handleQueryInput(String input) {
                 Map<Integer, Long> positionToIdMapping = controller.presentAllTasksForUserSelection();
                 if (positionToIdMapping.size() != 0) {
                     TaskInfo taskInfo = chooseTask(positionToIdMapping);
-                    controller.suggestTimeToUser(taskInfo);
-                    System.out.println("Event created from task");
-                }
-                break;
-            case "6":
-                positionToIdMapping = controller.presentAllTasksForUserSelection();
-                if (positionToIdMapping.size() != 0) {
-                    TaskInfo taskManual = chooseTask(positionToIdMapping);
+                    LocalDateTime suggestedTime = controller.getSuggestedTime(taskInfo.getDuration());
+                    String formattedTime = suggestedTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd-HH:mm"));
+                    System.out.println("Suggested time is: " + formattedTime);
+
```

Comment:
Can you add in a check for whether the user agrees with the suggested input?

Reply:
Great suggestion. I added a 'y' option.